src/ui/pages/Settings/Settings.tsx

```diff
@@ -149,6 +153,16 @@ function SettingsMain() {
                 </HStack>
               </AngleRightRow>
             </FrameListItemLink>
+            {hasTestWallet ? (
```

Comment:
This is minor, but having test wallet as a watched address also allows this item to appear. Do we want to avoid this?